kubernetes/README.md

```diff
@@ -0,0 +1,35 @@
+# Creating a Kubernetes Deployment
+
+## Instructions
+
+Copy your private key which you have configured with JIRA into the
+current directory.
+
+Fill out the config.yaml and secret.yaml file with the correct
+configuration. These are the same keys which are used in the application
+command line parameters or configuration file, as noted in the main
+README. Make sure they are base64 encoded, for example with
```

Comment:
"Make sure the values in the secret are base64-encoded..."

only values in the secret need to be base64 encoded.